test/virtual/enclave.cpp

```diff
@@ -84,6 +94,20 @@ extern "C"
             reinterpret_cast<uint8_t*>(output_buffer),
             output_buffer_size,
             output_bytes_written);
+
+        /* Emulate the args->deepcopy_out_buffer passing. */
+        oe_call_args_t* args = static_cast<oe_call_args_t*>(output_buffer);
+        if (args->deepcopy_out_buffer && args->deepcopy_out_buffer_size)
+        {
+            uint8_t* enclave_buffer =
+                (uint8_t*)oe_malloc(args->deepcopy_out_buffer_size);
+            memcpy(
+                enclave_buffer,
+                args->deepcopy_out_buffer,
+                args->deepcopy_out_buffer_size);
+            free(args->deepcopy_out_buffer);
```

Comment:
This needs to be oe_free.

Reply:
The behavior for OCALL is that 
Host allocates `args->deepcopy_out_buffer` (host buffer).
-> Enclave allocates an in-enclave buffer (`enclave_buffer`)
-> Enclave frees the original host buffer via `free`.
-> Have the `args->deepcopy_out_buffer` points to the `enclave_buffer`.

Here, we simply use `free` to simulate the `oe_host_free`.